src/Simplex/Chat.hs

```diff
@@ -4489,6 +4517,21 @@ processAgentMessageConn user@User {userId} corrId agentConnId agentMessage = do
           groupMsgToView g' m ci msgMeta
         createGroupFeatureChangedItems user cd CIRcvGroupFeature g g'
 
+    xGrpDirectInv :: GroupInfo -> GroupMember -> ConnReqInvitation -> Maybe MsgContent -> RcvMessage -> MsgMeta -> m ()
+    xGrpDirectInv g m connReq mContent_ msg msgMeta = do
+      -- check direct messages preference
+      -- auto-join / client setting? (new to_join field if auto join is in UI?)
+      -- create contact (PendingMemberContact?)
+      -- if member already has contact, replace it (to not mix messages from connections)
+      -- create chat item for contact
+      -- CRNewMemberContactInvitation
+      -- how to display "contact invitation" in UI?
```

Comment:
I thought as a special message in the group? But then maybe also use contact request view in the list as well?